app/routes/Event.tsx

```diff
@@ -0,0 +1,8 @@
+import Calendar from '~/components/Calendar.client';
+
+//TODO - create ticket to show spinner as a fallback component(Ticket will be attached)
```

Comment:
@harshith-venkatesh can we create ticket for this? A simple spinner can be shown here.

Reply:
Sure, I will go ahead and raise a ticket.
You could go ahead and remove the comment please